cpp/src/DataStorm/TraceUtil.cpp

```diff
@@ -32,3 +32,91 @@ TraceLevels::TraceLevels(const PropertiesPtr& properties, LoggerPtr logger)
 #elif defined(__GNUC__)
 #    pragma GCC diagnostic pop
 #endif
+
+// Our custom implementation of various operator<< for structs defined in Contract.ice
+
+namespace DataStormContract
```

Comment:
It would be nice if these were in the same order as the Slice definitions, instead of a seemingly random order.
The Slice order:
```
struct DataSamples
struct ElementInfo
struct TopicInfo
struct TopicSpec
struct ElementData
struct ElementSpec
struct ElementDataAck
struct ElementSpecAck
```

Reply:
Nevermind, I didn't realize this was just moved from `TraceUtil.h`.